### DISCUSSION TITLE
Multiple ``.csproj`` files for Razor Class libraries.

### DISCUSSION THREAD

**Discussion by AmSmart** [microsoft/dotnet-podcasts#9] [Q&A] [answered]
I am opening an issue for this question because this repo does not support discussions as of the time of writing.  I noticed that there was an extra ``csproj`` file created for the ``RCL``s shared between MAUI and Blazor. In my understanding, the extra ``.csproj`` file was to add the MAUI targets for the same project. I am confused as to why this is necessary since the RCL already targets .NET 6, I'd be glad if anyone can help point out what I seem to be missing or misunderstanding.

**Comment by AmSmart** [comment]
Stumbled on this issue - dotnet/maui#3288. Could it be the reason for this?

**Comment by jamesmontemagno** [comment, marked as answer]
Yup! That is the current reason. The csproj for RCLs in MAUI apps need the multi-targeting, but once that is fixed up it will be updated in the repo. As you can see they reference all of the same files.

**Comment by AmSmart** [comment]
That totally makes sense. Thanks, James. I guess I should leave the issue open in that case.
